src/main/java/calculator/WebCalculatorFactory.java

```diff
@@ -0,0 +1,10 @@
+package calculator;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class WebCalculatorFactory {
+    public Calculator createCalculator(HttpServletRequest req, HttpServletResponse resp) {
```

Comment:
Why does WebCalculatorFactory  shoulld know about HttpServletRequest HttpServletResponse ?
It would be better to send it as parameter in method 
calculator.calculate(req, resp);